src/Behat/Cores/Drupal8.php

```diff
@@ -469,4 +469,23 @@ public function createFileUrl($file, bool $relative = TRUE) {
     throw new InvalidArgumentException('%s method only accept %s objects in Drupal 8 or higher', __METHOD__, FileInterface::class);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getLanguagePrefix($language) {
+    $language_manager = \Drupal::languageManager();
+    $language_list = $language_manager->getStandardLanguageList();
+    array_walk($language_list, function (&$item1, $key) {
```

Comment:
The process of finding the value seems a little bit complex (walk over elements, flip and check for the language). I would use arra_filter and chek for results.

Adding the two comments below the function would be:
```
function getLanguagePrefix($language) {
  $language_manager = \Drupal::languageManager();
  $language_list = $language_manager->getStandardLanguageList();

  $filter_func = function($item) use ($language) { 
    return $language === $item[0];
  };
  
  $found = array_filter($language_list, $filter_func);
  
  if (empty($found)) {
    throw new \InvalidArgumentException(sprintf("Language %s not found", $language));  
  }

  $prefixes = \Drupal::config('language.negotiation')->get('url.prefixes');
  return $prefixes[array_key_first($found)];
}

```

Reply:
I would change this
```
    return $language === $item[0];
```

by this:

```
    return in_array($language, $item);
```

In this comment I explain why: https://github.com/Metadrop/behat-contexts/pull/125#discussion_r778057004